endpointmanager/pkg/endpointmanager/postgresql/npiorganizationstore.go

```diff
@@ -0,0 +1,116 @@
+package postgresql
+
+import (
+	"encoding/json"
+
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/endpointmanager"
+)
+
+// GetNPIOrganization gets a NPIOrganization from the database using the database id as a key.
+// If the NPIOrganization does not exist in the database, sql.ErrNoRows will be returned.
+func (s *Store) GetNPIOrganization(id int) (*endpointmanager.NPIOrganization, error) {
```

Comment:
any interest in being able to retrieve by `NPI_ID`?

Reply:
yes